src/mocks/handlers.ts

```diff
@@ -56,6 +60,10 @@ const mockGpa = { ...gpaHookData, data: gpaUndergraduateData };
 const { academicCalendar6, employeeEvents, studentExperienceEvents } = Events.mockEvents;
 const { academicAnnouncementResult } = Announcements.mockAnnouncements;
 const { resourcesCardData } = Resources.mockResources;
+const { resourcesData } = Resources.mockResources;
+const { favoriteResource } = Resources.mockResources;
+const { categoriesData } = Resources.mockResources;
+const { trendingResourcesData } = Resources.mockResources;
```

Comment:
In situations like this, you should destructure in a single line;

`const { resourcesData, favoriteResource, categoriesData, trendingResourcesData } = Resources.mockResources;`